src/fuzzfetch/fetch.py

```diff
@@ -250,7 +249,7 @@ def iterall(cls, build, branch, flags, platform=None):
                 build = HgRevision(build, branch).hash
             flag_str = flags.build_string()
             task_paths = tuple(path + flag_str for path in cls._revision_paths(build.lower(), branch, target_platform))
-            task_template_paths = itertools.product(cls.TASKCLUSTER_APIS, task_paths)
+            task_template_paths = itertools.product((cls.TASKCLUSTER_API,), task_paths)
```

Comment:
Since we only have the one template now, why bother using it in the loop (line 269)? We should just replace `template` with `cls.TASKCLUSTER_API` as you did in the other loop you eliminated on line 305.